aiorospy/src/aiorospy/action.py

```diff
@@ -12,6 +12,7 @@
 logger = logging.getLogger(__name__)
 
 
+# TODO(pbovbel) remove simple clients
```

Comment:
I'm in support of this. But why? Are they fundamentally broken or just something we don't want to support/encourage?

Reply:
a little of column A, a little of column B. Outside of the issues we've seen with SimpleActions (a per the behavior engine and move_base misadventures with LOST goals), IMO it would just be easier to encourage/maintain a single wrapper rather than multiple.

Reply:
Simple Action Clients have been a bit messed up for quite a while.  All for eliminating them here.